README.md

```diff
@@ -114,29 +118,34 @@ Promise.resolve(32).then(x => console.log(x)); // => 32
 - [Changelog](./CHANGELOG.md)
 
 ## Usage
-### Installation:
-```
-// global version
+
+### Installation
+
+```sh
+# global version
 npm install --save core-js@3.7.0
-// version without global namespace pollution
+# version without global namespace pollution
 npm install --save core-js-pure@3.7.0
-// bundled global version
+# bundled global version
```

Comment:
Why it's changed? Please, remove all changes unrelated to `exclude` from this PR.